glumpy/app/window/backends/backend_qt5.py

```diff
@@ -323,19 +323,7 @@ def _modifiers_translate( self, modifiers ):
         if QtCore.Qt.ShiftModifier & modifiers:
             _modifiers |= window.key.MOD_SHIFT
         if QtCore.Qt.ControlModifier & modifiers:
-            _modifiers |= window.key.MOD_CONTROL
-        if QtCore.Qt.AltModifier & modifiers:
-            _modifiers |= window.key.MOD_ALT
-        if QtCore.Qt.MetaModifier & modifiers:
-            _modifiers |= window.key.MOD_META
-        return _modifiers
-
-    def _modifiers_translate( self, modifiers ):
```

Comment:
Why did you remove this function ?

Reply:
Removed because it is a duplicate; _modifiers_translate is in there twice, and it had the exact same method body.